besom-cfg/lib/src/main/scala/Configured.scala

```diff
@@ -3,6 +3,11 @@ package besom.cfg
 import scala.quoted.*
 import besom.json.*
 import besom.cfg.internal.*
+import besom.util.Validated
+import scala.util.control.NoStackTrace
+import org.checkerframework.checker.units.qual.A
```

Comment:
I believe this line is still to be removed

Reply:
what the hell, I did kill it

Reply:
ok, pushed fix